mdapi/__init__.py

```diff
@@ -21,6 +21,56 @@
 of Red Hat, Inc.
 """
 
+import os
+import sys
 from importlib import metadata
+from logging import getLogger
+from logging.config import dictConfig
+
+from requests.packages.urllib3 import disable_warnings
+
+from mdapi.confdata.servlogr import logrobjc  # noqa
+from mdapi.confdata.standard import (  # noqa
+    APPSERVE,
+    CRON_SLEEP,
```

Comment:
I believe cron job would be a different entity independent of the deployment. If combined, this resemble extra redundancy in deployment.

Reply:
The included CLI application has two commands. One for serving the web application and the other one for populating the database files. Both of these commands would run separately from each other but would be invoked from the same CLI application.